### PR TITLE
Loc update

### PR DESCRIPTION
**Pull request by cristianosuzuki77**

### CODE REVIEW

File: src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf [hunk 1]
```diff
@@ -134,7 +134,7 @@
       </trans-unit>
       <trans-unit id="EnableAnalyzerReleaseTrackingRuleDescription">
         <source>Enabling release tracking for analyzer packages helps in tracking and documenting the analyzer diagnostics that ship and/or change with each analyzer release. See details at https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md.</source>
-        <target state="translated">L'activation du suivi de version pour les packages d'analyseurs permet de suivre et de documenter les diagnostics d'analyseurs qui sont fournis et/ou changés avec chaque version d'analyseur. Consultez les détails sur https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md.</target>
+        <target state="translated">L'activation du suivi de version pour les packages d'analyseurs permet de suivre et de documenter les diagnostics d'analyseurs qui sont fournis et/ou changés pour chaque version d'analyseur. Consultez les détails sur https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md.</target>
```
Comment (Youssef1313) [inline src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf:137]: @Evangelink Can you confirm the correctness of the change?

Comment (Evangelink) [reply src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf:137]: LGTM! IMO the previous version was fine too and is probably what I would have written.

File: src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf [hunk 2]
```diff
@@ -259,22 +259,22 @@
       </trans-unit>
       <trans-unit id="PreferIsKindDescription">
         <source>Prefer 'syntax.IsKind(kind)' to 'syntax.Kind() == kind' when checking syntax kinds. Code using 'IsKind' is slightly more efficient at runtime, so consistent use of this form where applicable helps improve performance in complex analysis scenarios.</source>
-        <target state="new">Prefer 'syntax.IsKind(kind)' to 'syntax.Kind() == kind' when checking syntax kinds. Code using 'IsKind' is slightly more efficient at runtime, so consistent use of this form where applicable helps improve performance in complex analysis scenarios.</target>
+        <target state="translated">Utilisez 'syntax.IsKind(kind)' au lieu de 'syntax.Kind() == kind' pour la vérification des genres de syntaxe. Dans la mesure où le code utilisant 'IsKind' est légèrement plus efficace au moment de l'exécution, l'utilisation cohérente de ce formulaire permet d'améliorer les performances dans les scénarios d'analyse complexes.</target>
```
Comment (Evangelink) [inline src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf:262]: There is a problem here, `formulaire` for `form` is one possible translation but not the correct one in this context. Also, `cohérente` for `consistent` is not the best translation here.


```suggestion
        <target state="translated">Utilisez 'syntax.IsKind(kind)' au lieu de 'syntax.Kind() == kind' pour la vérification des genres de syntaxe. Dans la mesure où le code utilisant 'IsKind' est légèrement plus efficace au moment de l'exécution, l'utilisation systématique de cette forme permet d'améliorer les performances dans les scénarios d'analyse complexes.</target>
```

or

```suggestion
        <target state="translated">Utilisez 'syntax.IsKind(kind)' au lieu de 'syntax.Kind() == kind' pour la vérification des genres de syntaxe. Dans la mesure où le code utilisant 'IsKind' est légèrement plus efficace au moment de l'exécution, l'utilisation systématique de ce dernier permet d'améliorer les performances dans les scénarios d'analyse complexes.</target>
```

File: src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.fr.xlf [hunk 3]
```diff
@@ -244,7 +244,7 @@
       </trans-unit>
       <trans-unit id="OperatorOverloadsHaveNamedAlternatesCodeFixTitle">
         <source>Add operator overload named alternate</source>
-        <target state="new">Add operator overload named alternate</target>
+        <target state="translated">Ajouter une surcharge d'opérateur nommée alternate</target>
```
Comment (Evangelink) [inline src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.fr.xlf:247]:
```suggestion
        <target state="translated">Ajouter une surcharge d'opérateur nommée alternative</target>
```

File: src/Text.Analyzers/Core/xlf/TextAnalyzersResources.fr.xlf [hunk 4]
```diff
@@ -49,37 +49,37 @@
...
       <trans-unit id="IdentifiersShouldBeSpelledCorrectlyMessageMethodTypeParameter">
         <source>On method '{0}', correct the spelling of '{1}' in generic type parameter name '{2}'</source>
-        <target state="needs-review-translation">Dans la méthode {0}, corrigez l'orthographe de '{1}' dans le nom de paramètre de type générique {2}</target>
+        <target state="translated">Dans la méthode '{0}', corrigez l'orthographe de '{1}' dans le nom de paramètre de type générique '{2}'</target>
         <note />
       </trans-unit>
       <trans-unit id="IdentifiersShouldBeSpelledCorrectlyMessageAssemblyMoreMeaningfulName">
         <source>Consider providing a more meaningful name than assembly name '{0}'</source>
-        <target state="needs-review-translation">Indiquez éventuellement un nom plus significatif que le nom d'assembly {0}</target>
+        <target state="translated">Si possible, indiquez un nom plus significatif que le nom d'assembly '{0}'</target>
```
Comment (Evangelink) [inline src/Text.Analyzers/Core/xlf/TextAnalyzersResources.fr.xlf:82]: It's correct but I would have used

```suggestion
        <target state="translated">Envisagez de fournir un nom plus significatif que le nom d'assembly '{0}'</target>
```

Comment (Evangelink) [reply src/Text.Analyzers/Core/xlf/TextAnalyzersResources.fr.xlf:82]: Something similar could be used for the other translations of this file.

File: src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf (hunk 1 above)
Comment (Youssef1313) [reply src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.fr.xlf:137]: Well, if both are okay then never mind. I wanted to note as my *very very* basic knowledge of French that "with" means "avec", and it felt strange to change it to "pour".

Thanks for taking a look.

**Comment by cristianosuzuki77** [comment]
Feedback incorporated (399043)
The feedback, if incorporated by translators (given their guidelines), will be coming in a future cumulative PR (every other Friday).
You could either:
a. Finish this PR and the next cumulative PR will have it.
b. Reject this PR and wait until the next cumulative PR.

**Comment by Evangelink** [comment]
Thanks @cristianosuzuki77 ! @mavasani for the final call here.

**Comment by cristianosuzuki77** [comment]
Since this was opened, I uploaded the updates from the translators.

**Review by mavasani — APPROVED** [review]

**Pull request merged**